logger.js

```diff
@@ -100,7 +100,9 @@ function pinoLogger (opts, stream) {
           url = URL.parse(req.url)
         }
         if (url && url.pathname) {
-          shouldLogSuccess = !autoLoggingIgnorePaths.includes(url.pathname)
+          shouldLogSuccess = !autoLoggingIgnorePaths.find(ignorePath => {
+            return new RegExp(ignorePath).test(url.pathname)
+          })
```

Comment:
We should not be generating a new regexp for strings that are anot already regexp.

Reply:
@mcollina let me know if you prefer some other approach. I thought this would work as well and be a bit more explicit.

Reply:
This is taking user input (`url.pathname`) and creating a new regular expression from it. See https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS

Reply:
This has been already fixed.

Reply:
👍